back/src/models/oauth.model.ts

```diff
@@ -1,42 +1,47 @@
 // oauth.model.ts
 
-import { Sequelize, DataTypes, Model } from 'sequelize';
+import {Sequelize, DataTypes, Model} from 'sequelize';
 
-class Oauth extends Model {
-  
-  public id!: number;
-  public serviceId!: number;
-  public oauthtToken!: string;
-  public ownerID!: number;
+class OAuth extends Model {
 
-  public static initialize(sequelize: Sequelize) {
-    Oauth.init(
-      {
-        id: {
-          type: DataTypes.INTEGER,
-          primaryKey: true,
-          allowNull: false,
-        },
-        serviceId: {
-          type: DataTypes.INTEGER,
-          unique: true,
-          allowNull: false,
-        },
-        oauthtToken: {
-          type: DataTypes.STRING,
-          allowNull: true,
-        },
-        ownerID: {
-          type: DataTypes.INTEGER,
-          allowNull: true,
-        },
-      },
-      {
-        sequelize,
-        tableName: 'oauth',
-      }
-    );
-  }
+    public id!: number;
```

Comment:
Should be no empty line here
```suggestion
class OAuth extends Model {
    public id!: number;
```